### DISCUSSION TITLE
A comparison of Bayesian vs frequentist MMRMs

### DISCUSSION THREAD

**Discussion by wlandau** [PSIAIMS/CAMIS#151] [Q&A]
I lead the Bayesian MMRM sub-team in [`openstatsware`](https://www.openstatsware.org), and we are working the [`brms.mmrm`](https://github.com/openpharma/brms.mmrm) package. We have three package vignettes to support validation: a [simulation-based calibration workflow](https://openpharma.github.io/brms.mmrm/articles/sbc.html), a [comparison with the `mmrm` package on a small dataset](https://openpharma.github.io/brms.mmrm/articles/fev1.html), and a [comparison with the `mmrm` package on a larger dataset](https://openpharma.github.io/brms.mmrm/articles/bcva.html). The last two feature the same datasets that [`mmrm` uses to compare itself with SAS](https://openpharma.github.io/mmrm/latest-tag/articles/mmrm_review_methods.html).

The sub-team is interested in contributing to CAMIS. However, in view of the current table at https://psiaims.github.io/CAMIS/#repository, our situation seems unusual. Our method is Bayesian and we are comparing against the `mmrm` R package instead of SAS (since `mmrm` and SAS already agree so closely). Is CAMIS the right place to share our method and validation materials?

**Comment by DrLynTaylor** [comment]
Hi @wlandau, Thanks so much for posting the above.    @andreaczhang  has reached out to you as we'd love to collaborate.
